ecs-cli/modules/cli/image/image_app.go

```diff
@@ -150,6 +158,18 @@ func pushImage(c *cli.Context, rdwr config.ReadWriter, dockerClient dockerclient
 		return err
 	}
 
+	if registryURI == "" {
```

Comment:
Would also be helpful to explain here in a comment that the full registry ARN is needed for tagging purposes.